apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -343,14 +349,14 @@ export class TriggerEvent {
         'no_subscriber_phone',
         'Subscriber does not have active phone'
       );
-    } else if (!application.channels?.sms?.twillio?.authToken) {
-      this.createLogUsecase.execute(
+    } else if (!command.payload.$fromPhone) {
+      await this.createLogUsecase.execute(
         CreateLogCommand.create({
           transactionId: command.transactionId,
           status: LogStatusEnum.ERROR,
           applicationId: command.applicationId,
           organizationId: command.organizationId,
-          text: 'No sms provider was configured',
+          text: 'Subscriber does not have active origin phone',
```

Comment:
The origin phone is not really related to the subscriber itself but to the application. The subscriber-only have a $phone.
```suggestion
          text: 'Integration was not configured with a from phone number',
```